mne/source_estimate.py

```diff
@@ -3056,9 +3079,43 @@ def _get_ico_tris(grade, verbose=None, return_surf=False):
         return ico
 
 
+@deprecated("This function is deprecated and will be removed in version 0.18. "
+            "Use instead stc.as_volume or stc.save_as_volume methods.")
 def save_stc_as_volume(fname, stc, src, dest='mri', mri_resolution=False):
     """Save a volume source estimate in a NIfTI file.
 
+    This function is DEPRECATED.
```

Comment:
You shouldn't need to add this because the decorator does something similar IIRC